fuzz/fuzz_targets/rtt_dag.rs

```diff
@@ -15,14 +15,15 @@
 extern crate simplicity;
 
 use simplicity::bititer::BitIter;
-use simplicity::core::TermDag;
-use simplicity::decode;
+use simplicity::core::{TermDag, UntypedProgram};
 use simplicity::jet::application::Core;
 
 fn do_test(data: &[u8]) {
     let mut iter = BitIter::new(data.iter().cloned());
 
-    if let Ok(program) = decode::decode_program_no_witness::<_, Core>(&mut iter) {
+    if let Ok(program) = UntypedProgram::<_, Core>::decode(&mut iter) {
+        // println!("{:?}", program);
```

Comment:
In d13f0e4aaa49162d46622b3c51b99198cfdc3530:

Not sure if you meant to commit this println (or the other one below). It's not a problem, just making you aware.

Reply:
This is how I debug: I uncomment a print statement of a decoded byte string that fails later down the line. Some fuzz tests already contained these comments, so I added the remaining ones.

Reply:
Sounds good!